apiaudio/__init__.py

```diff
@@ -18,8 +17,9 @@
 enable_telemetry = True
 max_network_retries = 0
 
-# Set to either 'debug' or 'info', controls console logging
-log = None
-
 # API resources
 from apiaudio.api_resources import *
+
+# logging
+from apiaudio.logging import SDKLogger
+APILogger = SDKLogger("apiaudio")
```

Comment:
I'm thinking one call during initiation? A specific endpoint that we can use to log user activity and/or return whatever updates we want?

Reply:
I think we might have to be careful about sending back any information collected on the client-side (GDPR and whatnot)